packages/test-commons/src/generateIsomorphicTests.ts

```diff
@@ -55,6 +57,24 @@ export function generateIsomorphicTests(
                         Enzyme.render(element);
                     });
                 }
+                if (classNameSkipList.includes(componentName)) {
+                    it.skip(`<${componentName}>`);
+                } else {
+                    it(`<${componentName}>  className is applied to outer element`, () => {
+                        const element = React.createElement(
+                            Component,
+                            {
+                                ...props[componentName],
+                                className: "test",
+                                iconName: "pt-icon-build",
+                                inline: true,
+                                lazy: false,
```

Comment:
why are these three props necessary? should only have to set `className`, other required props should come from `props` arg itself.

Reply:
Moved these to the isotest.js files so they will only be used for the components that need them